codegen/api-specs/nuts-storage-api-v1.yaml

```diff
@@ -5,25 +5,66 @@ info:
     API specification for a generic secret store HTTP server. 
     It can be use to implement a secure store or a proxy to a storage solution such as 
     Hashicorp Vault or Azure key store. Usually the Nuts node will be the client.
-    
-    The endpoints mounted at /secret/{path} is a key-value store which accepts keys as path.
   version: 1.0.0
   license:
     name: GPLv3
 servers:
   - url: http://localhost:7863
 paths:
-  /secrets/{path}:
+  /health:
+    get:
+      operationId: healthCheck
+      summary: Health check
+      description: |
+        This endpoint provides a healthcheck for the storage server.
+        It can be used by clients to check the availability of the services or the proxied storage solution.
+        The server should return a 200 status code if the service is available, and a 503 status code if the service is unavailable.
+      responses:
+        200:
+          description: |
+            The service is healthy
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ServiceStatus"
+        503:
+          description: |
+            The service is not healthy
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ServiceStatus"
+  /secrets:
+    get:
+      operationId: listKeys
+      summary: List all keys in the store
+      description: |
+        List all the keys available in the store.
```

Comment:
It lists the Key IDs and not the actual keys? Should be clear from description.

Reply:
It's a key/value store, were the values are a secret. So a key is always the ID of the secret. Listing the keys does not list the values.